Cargo.toml

```diff
@@ -41,7 +41,7 @@ alloy-transport-ipc = { version = "0.1.0", default-features = false, path = "cra
 alloy-transport-ws = { version = "0.1.0", default-features = false, path = "crates/transport-ws" }
 
 alloy-core = { version = "0.6.3", default-features = false, features = ["std"] }
-alloy-dyn-abi = { version = "0.6.3", default-features = false, features = ["std"] }
+alloy-dyn-abi = { version = "0.6.3", default-features = false, features = ["std", "eip712"] }
```

Comment:
can remove this and address it in the signers

Reply:
Resolved, thanks.